stackslib/src/burnchains/bitcoin/indexer.rs

```diff
@@ -837,7 +837,11 @@ impl BitcoinIndexer {
                 }
             } else {
                 // ignore the reorg
-                test_debug!("Reorg chain does not overtake original Bitcoin chain");
+                test_debug!(
+                    "Reorg chain does not overtake original Bitcoin chain ({} >= {})",
+                    orig_total_work,
+                    reorg_total_work
+                );
```

Comment:
Might be useful to upgrade this to `debug!`

Reply:
This is actually pretty chatty.  It'll get printed each time the node tries to sync with Bitcoin, because it looks for reorgs each time.

Reply:
ok, reverting.